### ISSUE TITLE
Attribute Table - Issue generating charts

### ISSUE THREAD

**Issue by tdipisa** [geosolutions-it/MapStore2#6187]
## Description

As soon as you select the aggregation method the chart disappear from the preview and it is not present in the final chart widget.

## How to reproduce

-  Open [this map](https://qa.mapstore2.geo-solutions.it/mapstore/#/viewer/openlayers/10358)
- Follow the steps reported in the GIF below

![fg_edit_6](https://user-images.githubusercontent.com/1280027/99049348-050d9e00-2597-11eb-89aa-dc0696d9f643.gif)

*Expected Result*

The chart should be available in preview and in the final widget according to the configured parameter **including the applied filters**

*Current Result*

The chart is no longer available as soon as you select the aggregation method

- [x] Not browser related

<details><summary> <b>Browser info</b> </summary>

(use this site: <a href="https://www.whatsmybrowser.org/">https://www.whatsmybrowser.org/</a> for non expert users)

| Browser Affected | Version |
|---|---|
|Internet Explorer| |
|Edge| |
|Chrome| |
|Firefox| |
|Safari| |
</details>

**Comment by tdipisa** [comment]
@ElenaGallo this issue should be fixed by #6077

**Comment by ElenaGallo** [comment]
@AGMETEOR here some notes:

- [ ] The selection remains visible on the map even after the widgets tool have been closed
![1featurewidget](https://user-images.githubusercontent.com/56537133/99286481-39a08480-2839-11eb-9b78-e28456d9e184.gif)

**How to reproduce**
- Open [this map](https://dev.mapstore2.geo-solutions.it/mapstore/#/viewer/openlayers/31145)
- Open the Attribute table of a layer
- Enable the Filter on map
- Select some features on map
- Click on widget
- Edit the widget
- Click on Save

*Current Result*
The features no longer need to be highlighted after the creation of the widget

- [ ] Selecting the feature one by one (via CTRL + click) it is not possible to create widgets
![2featurewidget](https://user-images.githubusercontent.com/56537133/99286492-3e653880-2839-11eb-893f-d6d76d23e3a7.gif)

**How to reproduce** 
- Open [this map](https://dev.mapstore2.geo-solutions.it/mapstore/#/viewer/openlayers/31145)
- Open the Attribute table of a layer
- Enable the Filter on map
- Select some features on map with the CTRL + click
- Click on widget
- Edit the widget

*Current Result*
The widget shows all features instead of just the selected ones

**Comment by tdipisa** [comment]
@AGMETEOR few more notes in addition to what already reported by @ElenaGallo above.

1.
>  The selection remains visible on the map even after the widgets tool have been closed

The selection on map should be visible only when the Attribute Table is open

2.
The issue I've reported in the description is still present if I select just one feature in map

![fg_edit_8](https://user-images.githubusercontent.com/1280027/99369545-db1cea00-28bc-11eb-939a-dff3b3d7dbef.gif)

**Comment by offtherailz** [comment]
Note: #6220 solves the issue with selection. The filter issue for the chart is present also starting from TOC and is a separate issue notified here:
https://github.com/geosolutions-it/MapStore2/issues/6221

**Comment by ElenaGallo** [comment]
Blocked by https://github.com/geosolutions-it/MapStore2/issues/6186#issuecomment-732182526
